src/routes/demos/save/index.svelte

```diff
@@ -101,9 +135,9 @@
       showModal = true;
     }}
     type="button"
-    class="sample button is-rounded is-warning is-light"
+    class="sample button is-rounded is-warning is-light px-5"
   >
-    Demo Sample
+    Preview Demo
   </button>
```

Comment:
The save button is still the primary CTA, while the "Preview" is the secondary/optional CTA. It might be better to visually indicate this by making the save button wider than the preview button. Also one would expect the save button to be on the right side of the preview button since that allows the user to "proceed" further.

Reply:
The preview button should be blue (info class)